src/Audit/Adapter/Database.php

```
@@ -0,0 +1,331 @@
+     * @param string $datetime
+     * @return bool
+     * @throws AuthorizationException|\Exception
+     */
+    public function cleanup(string $datetime): bool
+    {
+        $this->db->getAuthorization()->skip(function () use ($datetime) {
+            do {
+                $this->db->deleteDocuments($this->getCollectionName(), [
+                    Query::lessThan('time', $datetime),
+                ]);
+            } while (! empty($documents));
```

PHPMD (2.15.0)

Avoid unused local variables such as '$documents'. (undefined)

(UnusedLocalVariable)